content/en/users/tutorials/accessing-vm-with-ssh/_index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Accessing virtual machines with SSH"
+linkTitle: "Access a VM with SSH"
+type: docs
+weight: 20
+description: "Accessing virtual machines with SSH"
+---
+
+## Overview
+
+This page provides an introduction of connecting from a local computer to a
+cloud host via SSH. It provides general guidelines, SSH options, tips, and
+examples for setting up the OpenStack environment.
+
+## SSH Keys
+
+The recommended method to access a cloud virtual machine is via ssh using **SSH
+keys**, you may inject your **public key** into the virtual machine, at
+deployment time, and use your **private key** to connect via ssh without any
+password.
+
+{{% alert title="Tip" color="info" %}} If you are using ssh keys in GitHub your
+public keys are available at: `https://github.com/${github_username}.keys`.
+i.e.: `wget https://github.com/github_username.keys` {{% /alert %}}
+
+### SSH username
+
+The username to use to connect to a virtual machine is dependent on the virtual
+machine image and is generally different in each operative system image.
+
+For `official OS virtual machine images` you can use the general reference
```

Comment:
We are documenting the default user name in the AppDB, should we point to that instead?

Reply:
I've just added something to cover this as one of the possibilities, please have a look.